src/components/NewGalaxyModal.js

```diff
@@ -83,23 +99,29 @@ export const NewGalaxyModal = _.flow(
       }],
       ['paused', () => {
         return h(Fragment, [
-          h(ButtonPrimary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
-          h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: resumeGalaxy }, ['Resume'])
+          deleteButton(),
+          resumeButton()
         ])
       }],
       [Utils.DEFAULT, () => !app ?
-        h(ButtonPrimary, { onClick: () => setViewMode('createWarn') }, ['Next']) :
+        h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('createWarn') }, ['Next']) :
         Utils.switchCase(app.status,
           ['RUNNING', () => h(Fragment, [
-            deleteButton,
-            h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: () => { pauseGalaxy() } }, ['Pause']),
-            h(ButtonPrimary, { onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
+            deleteButton(),
+            pauseButton(),
+            h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
           ])],
           ['STOPPED', () => h(Fragment, [
-            deleteButton,
-            h(ButtonPrimary, { style: { marginRight: '1rem' }, onClick: () => { resumeGalaxy() } }, ['Resume'])
+            h(ButtonSecondary, { disabled: true, style: { marginRight: 'auto' }, tooltip: 'Cloud Compute must be resumed first.', onClick: () => setViewMode('deleteWarn') }, ['Delete']),
+            resumeButton()
           ])],
-          ['ERROR', () => deleteButton]
+          ['ERROR', () => deleteButton],
```

Comment:
Should this be `deleteButton()` (do parens matter?)

Reply:
Good catch! Parentheses matter in this case because the `deleteButton()` function has a parameter with a default value, which doesn't seem to get set if no `()`.